src/View.php

```diff
@@ -886,7 +909,7 @@ public function on($event, $selector = null, $action = null, $defaults = null)
             $this->js(true)->on($event, $action);
         }
 
-        return $thisAction;
+        return new jQuery();
```

Comment:
that's what $thisAction was :) not that i think i might have forgotten to add it into $actions[].